Snakefile

```diff
@@ -289,7 +289,23 @@ rule build_bus_regions:
         "scripts/build_bus_regions.py"
 
 
+def check_and_keep_cutout(config=config):
```

Comment:
Super! 🗡️ 

Sorry for being picky...
Do you mind adding a very small docstring to describe what this function does, something like "This function does this"

No need to describe inputs/output, very basic.
Moreover, the name of the function may be: "error_with_existing_cutouts" or something like that?

check and keep cutout seems to me a bit obscure but I may be wrong. Feel free to find better naming :)

Reply:
> Super! 🗡️
> 
> Sorry for being picky... Do you mind adding a very small docstring to describe what this function does, something like "This function does this"
> 
> No need to describe inputs/output, very basic. Moreover, the name of the function may be: "error_with_existing_cutouts" or something like that?
> 
> check and keep cutout seems to me a bit obscure but I may be wrong. Feel free to find better naming :)

Great point :D Thanks a lot for reviewing!
Fixed :)